src/Groups/action.jl

```diff
@@ -392,6 +386,49 @@ end
 ^(I::MPolyIdeal, p::MatrixGroupElem) = on_indeterminates(I, p)
 
 
+# We do not support `on_lines(line::Vector, x::GAPGroupElem)`
+# because for example `*(::fpFieldElem, ::Vector{fpFieldElem})`
+# is not supported.
+"""
+    on_lines(line::GAP.GapObj, x::GAPGroupElem)
+    on_lines(line::AbstractAlgebra.Generic.FreeModuleElem, x::GAPGroupElem)
+
+Return the image of the nonzero vector `line` under `x`,
+where the action is given by first computing `line * x`
+and then normalizing the result by scalar multiplication from the left
+such that the first nonzero entry is the `one` of the `base_ring` of `line`.
+
+# Examples
+```jldoctest
+julia> n = 2;  F = GF(5);  g = general_linear_group(n, F);
+
+julia> v = gen(free_module(F, n), 1)
+(1, 0)
+
+julia> m = gen(g,2)
```

Comment:
```suggestion
julia> m = gen(g, 2)
```
for consistency with the rest of the code in this docstring?